modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/QuantileCalculationMethod.java

```diff
@@ -23,34 +23,16 @@
    * If index value computed from the level is outside of the sample data range, 
    * {@code IllegalArgumentException} is thrown. 
    * <p> 
-   * The sample observations are sorted from the smallest to the largest. 
-   * 
-   * @param level  the quantile level
-   * @param sortedSample  the sample observations
-   * @return the quantile estimation
-   */
-  public double quantileFromSorted(double level, DoubleArray sortedSample) {
```

Comment:
We probably want to keep the existing methods and add new ones. Beyond backward compatibility, there is value in keeping those simpler methods returning simply a double (from sorted or notSorted). I would not remove the exiting methods.